linera-core/src/unit_tests/test_utils.rs

```diff
@@ -321,7 +321,7 @@ where
         blobs: Vec<Blob>,
     ) -> Option<Result<ChainInfoResponse, NodeError>> {
         match validator.fault_type {
-            FaultType::DontProcessValidated if certificate.value().is_validated() => None,
+            FaultType::DontProcessValidated if certificate.inner().is_validated() => None,
```

Comment:
Would it make sense to keep the term `value` here rather than `inner`? It isn't the inner _certificate_, after all, but rather the value that this is a certificate _of_.

Reply:
But would it be better the other way round then?

Reply:
TBH I prefer the current version - `inner()` returning what's _inside_ the `Hashed<_>` while `value()` returning `value` field.